src/plugins/noMouseNavigation/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "NoMouseNavigation",
+    description: "Aka FFMSB. Disables forward/back navigation with mouse side buttons",
```

Comment:
only very little users will know what FFMSB stands for. you can add it via the `tags:` property instead
```suggestion
    description: "Disables forward/back navigation with mouse side buttons",
```

Reply:
Agreed, changed according to suggestion